src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfFeature.java

```diff
@@ -0,0 +1,1453 @@
+        return baseData.annotationSource;
+    }
+
+    public FeatureType getFeatureType() {
+        return baseData.featureType;
+    }
+
+    public int getGenomicStartLocation() {
+        return baseData.genomicPosition.getStart();
+    }
+
+    public int getGenomicEndLocation() {
+        return baseData.genomicPosition.getEnd();
+    }
+
+    public GenomicStrand getGenomicStrand() {
+        return baseData.genomicStrand;
+    }
+
+    public GenomicPhase getGenomicPhase() {
+        return baseData.genomicPhase;
+    }
+
+    public String getGeneId() {
+        return baseData.geneId;
+    }
+
+    public String getTranscriptId() {
+        return baseData.transcriptId;
+    }
+
+    public GeneTranscriptType getGeneType() {
+        return baseData.geneType;
+    }
+
+    public String getGeneName() {
+        return baseData.geneName;
+    }
+
+    public GeneTranscriptType getTranscriptType() {
+        return baseData.transcriptType;
+    }
+
+    public String getTranscriptName() {
+        return baseData.transcriptName;
+    }
+
+    public GeneTranscriptStatus getGeneStatus() {
+        return baseData.geneStatus;
+    }
+
+    public GeneTranscriptStatus getTranscriptStatus() {
+        return baseData.transcriptStatus;
+    }
+
+    public int getExonNumber() {
+        return baseData.exonNumber;
+    }
+
+    public String getExonId() {
+        return baseData.exonId;
+    }
+
+    public LocusLevel getLocusLevel() {
+        return baseData.locusLevel;
+    }
+
+    public List<OptionalField<?>> getOptionalFields() {
+        return baseData.optionalFields;
+    }
+
+    public String getAnonymousOptionalFields() {
+        return baseData.anonymousOptionalFields;
+    }
+
+    public OptionalField<?> getOptionalField(String key) {
+        for (OptionalField<?> optionalField : baseData.optionalFields) {
+            if ( optionalField.getName().equals(key) ) {
+                return optionalField;
+            }
+        }
+        return null;
+    }
+
+    /**
+     * Comparable interface implementation for {@link GencodeGtfFeature}.
+     *
+     * Order is determined by {@link GencodeGtfFeatureBaseData#featureOrderNumber}
+     *
+     * @param other {@link GencodeGtfFeature} to which to compare
+     * @return -1 if this < other; 0 if this == other; 1 if this > other
+     */
+    @Override
+    public int compareTo(GencodeGtfFeature other) {
+        Utils.nonNull(other);
+        return (baseData.featureOrderNumber - other.baseData.featureOrderNumber);
+    }
+
+    @Override
+    public boolean equals(Object that) {
+        if (that == null) {
+            return false;
+        }
+        else if ( this == that ) {
+            return true;
+        }
+
+        boolean isEqual = that instanceof GencodeGtfFeature;
+        if (isEqual) {
+            GencodeGtfFeature thatFeature = (GencodeGtfFeature) that;
+            Objects.equals(baseData, thatFeature.baseData);
```

Comment:
BUGBUG: you need to assign the return value of `Objects.equals()` back to `isEqual`!

Reply:
Yup!  Good find!